<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_config_arboles">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>381</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>Config de arboles</string>
    </property>
    <widget class="QLabel" name="label_price">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rango de precio de fruta desde              a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_arbol">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Arbol:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_t_crec">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_cost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>costo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_t_crec">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiempo de crecimiento</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_arboles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ABB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AVL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RojiNegro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Heap</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cosecha">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_price">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>350</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_cosecha_x_t">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cosecha             frutos cada           segundos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_t_cosecha">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_price_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>350</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_config_plagas">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>Config de plagas</string>
    </property>
    <widget class="QLabel" name="label_plaga">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Plaga</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_plagas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Cuervo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Oveja</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Termitas</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prob">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_proba_comer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiene la probabilidad del             de aparecer cada            segundos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prob_x_t">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>290</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_come_x_t">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Come           frutos cada           segundos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_come">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_t_come">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Jugar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_guardar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
